windows-icon-handler/refresh-icon-handler/resource.h

```diff
@@ -0,0 +1,20 @@
+//{{NO_DEPENDENCIES}}

```

Comment:
Header ?

Reply:
This file is auto generated too. I'm wondering if we need to commit this file too. What do you think @Max-7 ?